src/features/user/MFV2/ForestProgress/microComponents/EmptyProgress.tsx

```diff
@@ -0,0 +1,22 @@
+import targetBarStyle from '../TreeTargetBar.module.scss';
+import { useTranslations } from 'next-intl';
+
+interface EmptyProgressProps {
+  handleOpen: () => void;
+}
+
+const EmptyProgress = ({ handleOpen }: EmptyProgressProps) => {
+  const tProfile = useTranslations('Profile');
+  return (
+    <div className={targetBarStyle.EmptyProgressContainer}>
+      <div className={targetBarStyle.setTargetLabel}>
+        {tProfile('progressBar.targetInfo')}
+      </div>
+      <button className={targetBarStyle.setTargetButton} onClick={handleOpen}>
+        {tProfile('progressBar.setTargets')}
+      </button>
```

Comment:
Replace this with WebappButton (so that we have consistent button styles)

e.g.
```
<WebappButton
      elementType="link"
      target="_blank"
      variant="tertiary"
      text={buttonText}
      href={donateLink}
      buttonClasses={buttonClasses}
    />